docs/data/material/components/progress/CustomizedProgressBars.tsx

```diff
@@ -54,12 +55,30 @@ function FacebookCircularProgress(props: CircularProgressProps) {
   );
 }
 
+// From https://github.com/mui/material-ui/issues/9496#issuecomment-959408221
+function GradientCircularProgress() {
+  return (
+    <React.Fragment>
+      <svg width={0} height={0}>
+        <defs>
+          <linearGradient id="my_gradient" x1="0%" y1="0%" x2="0%" y2="100%">
+            <stop offset="0%" stopColor="rgba(184, 74, 202, 1)" />
+            <stop offset="100%" stopColor="rgba(142, 66, 235, 1)" />
+          </linearGradient>
```

Comment:
Can you make the color's hue more different? I did not notice at first and I thought it was a bug.